Library/Homebrew/test/support/helper/spec/shared_examples/hbc_staged.rb

```diff
@@ -1,21 +1,17 @@
-require "spec_helper"
```

Comment:
I reckon this should be in a `cask` directory as `hbc` isn't obviously relating to Cask.